lib/assessment.js

```diff
@@ -5,18 +5,31 @@
 // its index, and the array itself
 // e.g. callback(currentValue, index, array)
 
+Matt or Saad: this has been a rough day for assessments, I will do whatever it takes to learn this stuff I just need some help.  I feel like I need to start over on the lesson for call backs then move into the four potential array methods.
+
```

Comment:
Do you want to resubmit it? I'd be happy to give you feedback on what you turn in.

Alternatively, we can just work through the solution together. Let me know what works for you!